readme.md

```diff
@@ -5,11 +5,11 @@ This repository contains documents and code used by cellxgene's curation team. I
 For information/issues about cellxgene and its portal please refer to:
 
 - [cellxgene](https://github.com/chanzuckerberg/cellxgene)
```

Comment:
That reference is for desktop cellxgene. It should be

`[single-cell-explorer](https://github.com/chanzuckerberg/single-cell-explorer)`